tests/java/CheckstyleBearTest.py

```diff
@@ -31,3 +32,13 @@ def test_with_custom_configfile(self):
         self.section["checkstyle_configs"] = self.empty_config
         self.check_validity(self.uut, [], self.good_file)
         self.check_validity(self.uut, [], self.bad_file)
+
+
+def test_known_checkstyle(monkeypatch):
```

Comment:
In that case you would do `from bears.java.CheckstyleBear import CheckstyleBear, known_checkstyle_or_path`